core/SettingsServer.php

```diff
@@ -91,7 +91,7 @@ public static function isIIS()
      */
     public static function isWindows()
     {
-        return DIRECTORY_SEPARATOR === '\\';
+        return PHP_OS_FAMILY === "Windows";
```

Comment:
Wondering if there might be any case where `Windows` could be reported as `Unknown`. Guess we could fall back to the old check in that case. But might be enough to add that as soon as someone reports an issue with it...

Reply:
I doubt many people use binaries apart from the official ones on Windows. But this might make sense for non-(Linux|BSD|MacOS) operating systems that Matomo probably should handle like Windows (don't assume unix tools exists).